src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -129,6 +129,19 @@ module Types = struct
     scalar "PublicKey" ~doc:"Base58Check-encoded public key string"
       ~coerce:(fun key -> `String (Public_key.Compressed.to_base58_check key))
 
+  let to_signed_fee_exn sign magnitude =
```

Comment:
I don't see this used anywhere else, except in `Client`. Can you put this in `Client.ml`?